metadata.rb

```diff
@@ -20,14 +20,15 @@
 # versions that someone has installed.
 gem 'aws-sdk-cloudformation', '~> 1.21.0'
 gem 'aws-sdk-cloudwatch', '~> 1.22.0'
-gem 'aws-sdk-core', '~> 3.52.0'
+gem 'aws-sdk-core', '~> 3.71.0'
 gem 'aws-sdk-dynamodb', '~> 1.28.0'
-gem 'aws-sdk-ec2', '~> 1.84.0'
+gem 'aws-sdk-ec2', '~> 1.115.0'
 gem 'aws-sdk-elasticloadbalancing', '~> 1.14.0'
 gem 'aws-eventstream', '~> 1.0.3'
 gem 'aws-sdk-iam', '~> 1.22.0'
 gem 'aws-sdk-kinesis', '~> 1.15.0'
 gem 'aws-sdk-kms', '~> 1.20.0'
+gem 'aws-sdk-resources', '~> 3.56.0'
```

Comment:
We don't want to pull in this dep since it's all 194 resource gems. If we need a particular version of ec2 and core that's handled above.

Reply:
I tested removing this one and it seems to pass CI.